creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -631,51 +892,37 @@ class SnapbackSM {
-        stage: 'buildSecondaryNodesToUserWalletsMap() Success',
-        vals: { numSecondaryNodes: Object.keys(secondaryNodesToUserWalletsMap).length },
-        time: Date.now()
-      })
-
-      // Retrieve clock statuses for all secondary users from secondary nodes
-      let secondaryNodesToUserClockStatusesMap
+      // Setup the mapping of Content Node endpoint to service provider id. Used in reconfig
       try {
-        secondaryNodesToUserClockStatusesMap = await this.retrieveClockStatusesForSecondaryUsersFromNodes(
-          secondaryNodesToUserWalletsMap
-        )
+        await this.peerSetManager.updateEndpointToSpIdMap(this.audiusLibs.ethContracts)
+        // Reset `this.highestReconfigModeEnabled` to the enabled mode
+        this.highestReconfigModeEnabled = this.getHighestReconfigModeEnabled()
+
         decisionTree.push({
-          stage: 'retrieveClockStatusesForSecondaryUsersFromNodes() Success',
-          vals: { },
+          stage: `updateEndpointToSpIdMap() Success`,
+          vals: {
+            endpointToSPIdMapSize: Object.keys(this.peerSetManager.endpointToSPIdMap).length
+          },
           time: Date.now()
         })
       } catch (e) {
+        // If unable to initialize `endpointToSpIdMap`, disable reconfig
+        this.highestReconfigModeEnabled = RECONFIG_MODES.RECONFIG_DISABLED.key
+
         decisionTree.push({
-          stage: 'retrieveClockStatusesForSecondaryUsersFromNodes() Error',
-          vals: e.message,
+          stage: `updateEndpointToSpIdMap() Error`,
+          vals: {
+            endpointToSPIdMapSize: Object.keys(this.peerSetManager.endpointToSPIdMap).length,
+            error: e.message
+          },
           time: Date.now()
         })
-        throw new Error('processStateMachineOperation():retrieveClockStatusesForSecondaryUsersFromNodes() Error')
       }
 
-      // Setup the mapping of Content Node endpoint to service provider id. Used in reconfig
-      const previousSnapbackReconfigEnabledValue = this.snapbackReconfigEnabled
-      await this.updateEndpointToSpIdMap()
-      decisionTree.push({
-        stage: `updateEndpointToSpIdMap()`,
-        vals: {
-          previousSnapbackReconfigEnabledValue,
-          currentSnapbackReconfigEnabledValue: this.snapbackReconfigEnabled,
-          endpointToSPIdMapSize: Object.keys(this.endpointToSPIdMap).length
-        },
-        time: Date.now()
-      })
-
       // Issue all required sync requests
       let numSyncRequestsRequired, numSyncRequestsEnqueued, enqueueSyncRequestErrors
       try {
-        const resp = await this.issueSyncRequests(potentialSyncRequests, secondaryNodesToUserClockStatusesMap)
+        const resp = await this.issueSyncRequests(potentialSyncRequests, replicaSetNodesToUserClockStatusesMap)
         numSyncRequestsRequired = resp.numSyncRequestsRequired
         numSyncRequestsEnqueued = resp.numSyncRequestsEnqueued
         enqueueSyncRequestErrors = resp.enqueueSyncRequestErrors
```

Comment:
nit - this is my fault but line 952 and 940 should be `numEnqueueSyncRequestErrors` not `numIssueSyncRequestErrors`

also my fault but the check one L952 can be simplified with line 940

Reply:
updatedd